diff -Naur --no-dereference u-boot-2021.04/drivers/phy/rockchip/Kconfig u-boot-2021.04-new/drivers/phy/rockchip/Kconfig
--- u-boot-2021.04/drivers/phy/rockchip/Kconfig	2021-04-05 23:03:29.000000000 +0800
+++ u-boot-2021.04-new/drivers/phy/rockchip/Kconfig	2021-06-24 19:00:08.322331300 +0800
@@ -11,6 +11,13 @@
 	help
 	  Support for Rockchip USB2.0 PHY with Innosilicon IP block.
 
+config PHY_ROCKCHIP_INNO_USB3
+	tristate "Rockchip INNO USB 3.0 PHY Driver"
+	depends on ARCH_ROCKCHIP
+	select PHY
+	help
+	  Support for Rockchip USB 3.0 PHY with Innosilicon IP block.
+
 config PHY_ROCKCHIP_PCIE
 	bool "Rockchip PCIe PHY Driver"
 	depends on ARCH_ROCKCHIP
diff -Naur --no-dereference u-boot-2021.04/drivers/phy/rockchip/Makefile u-boot-2021.04-new/drivers/phy/rockchip/Makefile
--- u-boot-2021.04/drivers/phy/rockchip/Makefile	2021-04-05 23:03:29.000000000 +0800
+++ u-boot-2021.04-new/drivers/phy/rockchip/Makefile	2021-06-24 16:36:23.330245500 +0800
@@ -4,6 +4,7 @@
 #
 
 obj-$(CONFIG_PHY_ROCKCHIP_INNO_USB2)	+= phy-rockchip-inno-usb2.o
+obj-$(CONFIG_PHY_ROCKCHIP_INNO_USB3)	+= phy-rockchip-inno-usb3.o
 obj-$(CONFIG_PHY_ROCKCHIP_PCIE)		+= phy-rockchip-pcie.o
 obj-$(CONFIG_PHY_ROCKCHIP_SNPS_PCIE3)	+= phy-rockchip-snps-pcie3.o
 obj-$(CONFIG_PHY_ROCKCHIP_TYPEC)	+= phy-rockchip-typec.o
